CHANGES.md

```diff
@@ -242,7 +242,7 @@ New Features
 ------------------
 - Added HMAC key management for service accounts. This includes a new "hmac"
   command to create/get/list/update/delete HMAC keys on service accounts.
-  Currently, your project must be whitelisted for use with this new HMAC
+  Currently, your project must be added to allowlist for use with this new HMAC
```

Comment:
What do you think about using 'allowlisted', or maybe 'allow-listed' instead of 'added to allowlist'? At the very least I think we should say 'added to _the_ allowlist'.

Reply:
Done. Thanks